pytest_inmanta/plugin.py

```diff
@@ -1130,3 +1142,16 @@ def finalize_handler(self, handler: ResourceHandler) -> None:
     def finalize_all_handlers(self) -> None:
         for handler_instance in self._handlers:
             self.finalize_handler(handler_instance)
+
+
+@pytest.fixture(scope="session")
+def session_temp_dir(tmpdir_factory: "TempdirFactory") -> Generator[str, None, None]:
+    session_temp_dir = tmpdir_factory.mktemp("session")
+    yield str(session_temp_dir)
+    session_temp_dir.remove(ignore_errors=True)
+
+
+@pytest.fixture
+def use_session_temp_dir(session_temp_dir: str) -> Generator[str, None, None]:
```

Comment:
I don't think this is a very good name for this method. `use_session_temp_dir` doesn't very well describe what the session temp dir is used for.